docs/how_to/noisy_simulators.rst

```diff
@@ -0,0 +1,429 @@
+Noisy Simulators in Qiskit Runtime
+==================================
+
+This notebook shows how to setup ``ibmq_qasm_simulator`` and map a basic
+noise model for an IBM Quantum hardware device in **Qiskit Runtime**, and
+use this noise model to do noisy simulations of ``QuantumCircuits`` on
+``Sampler`` and ``Estimator`` to study the effects of errors which occur
+on real devices.
+
+Set up your local development environment
+-----------------------------------------
+
+This tutorial requires a Qiskit Runtime service instance to be setup. If
+you haven’t done so already, please follow `these
+steps <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/getting_started.html>`__
+to set one up.
+
+.. code-block:: python
+
+    # load necessary Runtime libraries
+    from qiskit_ibm_runtime import QiskitRuntimeService, Sampler, Estimator, Session, Options
+
+    service = QiskitRuntimeService(channel="ibm_quantum")
+
+Preparing the environment
+-------------------------
+
+To demonstrate the routine, we shall proceed with running an example
+routine. One of the major benefits of using primitives is simplification
+of binding multiple parameters in parameterized circuits. To check this,
+here is an example circuit with a controlled
+`P-gate <https://qiskit.org/documentation/stubs/qiskit.circuit.library.PhaseGate.html>`__
+as implemented right below. Here, we parametrise the ``P-gate`` with a
+rotation parameter ``theta``. To learn how to create circuits and bind
+parameters to them by using Qiskit, see the `Circuit
+Basics <https://qiskit.org/documentation/tutorials/circuits/01_circuit_basics.html>`__
+and `Advanced
+Circuits <https://qiskit.org/documentation/tutorials/circuits_advanced/01_advanced_circuits.html#Parameterized-circuits>`__
+in Qiskit documentation.
+
+.. code-block:: python
+
+    from qiskit.circuit import Parameter
+    from qiskit import QuantumCircuit
+
+    theta = Parameter('theta')
+
+    qc = QuantumCircuit(2,1)
+    qc.x(1)
+    qc.h(0)
+    qc.cp(theta,0,1)
+    qc.h(0)
+    qc.measure(0,0)
+
+    qc.draw('mpl')
+
+
+
+
+.. image:: /docs/images/noisy-sim-circuit.png
```

Comment:
```suggestion
.. image:: ../images/noisy-sim-circuit.png
```

I think the docs build is failing because all the images need to be referenced like this

Reply:
Thank you @kt474 ! I've just changed every path of images! 😄

Reply:
one last thing, I think you'll have to add something like this to the doctree in index.rst [here](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/docs/index.rst) in the "How to" section

`Run noisy simulations <how_to/noisy_simulators>`

Reply:
I added this line in index.rst. Thanks @kt474 !